src/libs/Navigation/linkTo.js

```diff
@@ -23,9 +23,9 @@ export default function linkTo(navigation, path) {
 
     // Traverse up to get the root navigation
     // eslint-disable-next-line no-cond-assign
-    while ((current = root.dangerouslyGetParent())) {
-        root = current;
-    }
+    // while ((current = root.dangerouslyGetParent())) {
```

Comment:
This function is removed in the new version. I'm looking for alternatives. Any help appreciated here 👍

Reply:
I am sure we are using the native implementation of `useLinkTo` hook here. So you can take hints from there.

I guess we can use 
```
      // Traverse up to get the root navigation
      while ((current = root.getParent())) {
        root = current;
      }
```

Reply:
Thanks for the help, it worked 👍